packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -220,6 +232,59 @@ const Breakouts = WebexPlugin.extend({
 
     this.breakouts.set(Object.values(breakouts));
   },
+
+  /**
+   * Make enable breakout resource
+   * @returns {Promise}
+   */
+  touchBreakout() {
+    // @ts-ignore
+    return this.webex
+      .request({
+        method: HTTP_VERBS.POST,
+        uri: this.breakoutServiceUrl,
+        body: {
+          locusUrl: this.locusUrl,
+        },
+      })
+      .catch((err) => {
+        LoggerProxy.logger.error(
+          `Meeting:request#touchBreakout --> Error provisioning error ${err}`
+        );
+        throw err;
+      });
+  },
+
+  /**
+   * Make the meeting enbale or disable breakout session
+   * @param {boolean} enable
+   * @returns {Promise}
+   */
+  async toggleBreakout(enable) {
```

Comment:
Question: Do we care about a case where toggleBreakout() is called once and then again before the previous operation is fully completed and as a result we would call touchBreakout() a second time. What will happen then?

Reply:
Hmm, let me check it.

Reply:
I checked it, if call touchBreakout() again before the previous operation is fully completed, loucs will return the same object for the same call.